app/build.gradle

```diff
@@ -1,8 +1,9 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdk 35
     namespace "ai.elimu.analytics"
+    compileSdk 35
```

Comment:
@Federico-15 If you set the `compileSdk` to 35, also set the `targetSdkVersion` to 35.